### PR TITLE
Upsert fix

### LINKED ISSUE #7: upsert is not working
I tried use upsert(). But nothing is reflecting in database, However it is not generating even errors
```
db.upsert(table = SERVER_STORAGE_TABLE,
                 value = {'name':server, 'ping':'no'}
                  update_columns=['ping'])
```
Here is my code

### PR DESCRIPTION
**Pull request by sayanarijit**
#7 Small fix for upsert

### CODE REVIEW

File: dictmysql.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/python
 # -*-coding: utf-8 -*-
 
-from __future__ import print_function
```
Comment (sayanarijit) [inline dictmysql.py:4]: I didn't see any use of print function in this module, hence removed it.

File: dictmysql.py [hunk 2]
```diff
@@ -420,15 +419,16 @@ def upsert(self, table, value, update_columns=None, commit=True):
             update_columns = value.keys()
 
         value_q, _args = self._value_parser(value, columnname=False)
+        value_qs = dict(zip(update_columns, value_q.split(', ')))
```
Comment (sayanarijit) [inline dictmysql.py:422]: I hope splitting with comma won't create any problem.

File: dictmysql.py (hunk 1 above)
Comment (gyli) [reply dictmysql.py:4]: I will review the rest of this PR later, but print_function enables print() on Python 2.6. Due to the simplicity of this project, adding this line to add 2.6 support is worth.

Comment (sayanarijit) [reply dictmysql.py:4]: I know. But I checked print() is not used anywhere in this module.

**Comment by sayanarijit** [comment]
Hi, Just following up as it has been quite some time since this PR is opened. :slightly_smiling_face:

**Comment by gyli** [comment]
Hi,

I am sorry that I don't have a lot of time in the past month. I might be
able to take a look of it this weekend.

Thanks!

Best,
Ligyxy

**Comment by sayanarijit** [comment]
Sure no prob. Please let me know if I can do something to make it easier for review.

**Comment by gyli** [comment]
Please see my fix 5f7fb6e0276a3f082c95558f268cd6f1889b4782

**Pull request closed without merging**